### PR TITLE
Add ":" port colon to new IP:PORT method

### PR DESCRIPTION
**Pull request by systemcrash**
Should now insert a colon to separate IP and PORT params in call graphs

### CODE REVIEW

File: src/app/components/search-grid-call/detail-dialog/tab-flow/tab-flow.component.ts [hunk 1]
```diff
@@ -118,7 +118,8 @@ export class TabFlowComponent implements OnInit, AfterViewInit, OnDestroy {
             const regex = RegExp('(.*(?!$))(?::)([0-9]+)?$');
             if(regex.exec(i) != null){
                 const IP    = regex.exec(i)[1] // gives IP
-                const PORT  = regex.exec(i)[2] // gives port
+                var PORT  = regex.exec(i)[2] // gives port
```
Comment (lmangani) [inline src/app/components/search-grid-call/detail-dialog/tab-flow/tab-flow.component.ts:121]: Port should be an integer, so I guess perhaps the following might do?
`var PORT  = regex.exec(i)[2]  || 0 `

Comment (systemcrash) [reply src/app/components/search-grid-call/detail-dialog/tab-flow/tab-flow.component.ts:121]: 'should be'?

RegExp return strings (current code block works well).

Comment (systemcrash) [reply src/app/components/search-grid-call/detail-dialog/tab-flow/tab-flow.component.ts:126]: BTW if old and new methods prepend ":" to PORT, then it's pointless having it as an int...

Comment (lmangani) [reply src/app/components/search-grid-call/detail-dialog/tab-flow/tab-flow.component.ts:126]: I'm missing the point of this PR and what it wants to do or fix, unless you can refer it to an issue being discussed. Perhaps others will chime in.

Comment (systemcrash) [reply src/app/components/search-grid-call/detail-dialog/tab-flow/tab-flow.component.ts:126]: Compare with 'old' code:

https://github.com/sipcapture/homer-ui/blob/8f179c4e602020fd603b38e47d1a16f192d90d0e/src/app/components/search-grid-call/detail-dialog/tab-flow/tab-flow.component.ts#L127

Comment (lmangani) [reply src/app/components/search-grid-call/detail-dialog/tab-flow/tab-flow.component.ts:126]: The patch might as well be perfect, but I still don't understand what you're trying to fix Paul. Is this related to an open issue?

Comment (systemcrash) [reply src/app/components/search-grid-call/detail-dialog/tab-flow/tab-flow.component.ts:126]: Old code

IP:PORT
IPv6 <- fails

New code currently

IPPORT
IPv6PORT

This patch

IP:PORT
IPv6:PORT

Comment (systemcrash) [reply src/app/components/search-grid-call/detail-dialog/tab-flow/tab-flow.component.ts:126]: Consider it the finishing half of https://github.com/sipcapture/homer-ui/pull/227

**Pull request merged**